File: LocalPayment/src/main/java/com/braintreepayments/api/localpayment/LocalPaymentAuthResultInfo.java
package com.braintreepayments.api.localpayment;


import com.braintreepayments.api.BrowserSwitchFinalResult;

/**
 * Details of a {@link LocalPaymentAuthResult.Success}
 */
public class LocalPaymentAuthResultInfo {

    private final BrowserSwitchFinalResult.Success browserSwitchSuccess;

    LocalPaymentAuthResultInfo(BrowserSwitchFinalResult.Success browserSwitchSuccess) {
        this.browserSwitchSuccess = browserSwitchSuccess;
    }

    BrowserSwitchFinalResult.Success getBrowserSwitchResultInfo() {
        return browserSwitchSuccess;
    }
}

File: LocalPayment/src/main/java/com/braintreepayments/api/localpayment/LocalPaymentAuthResultInfo.kt
package com.braintreepayments.api.localpayment

import androidx.annotation.RestrictTo
import com.braintreepayments.api.BrowserSwitchFinalResult

/**
 * Details of a [LocalPaymentAuthResult.Success]
 */
@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
class LocalPaymentAuthResultInfo internal constructor(
    internal val browserSwitchSuccess: BrowserSwitchFinalResult.Success
)
